docs/components/PLACEHOLDERS.md

```diff
@@ -0,0 +1,87 @@
+# Placeholder component
+
+The Otter framework provides a placeholder component mechanism to help integrate dynamic HTML elements (with a basic rendering system) at a predefined position in the application.
+
+The placeholder component is exposed by the **@o3r/components** package and is working with the NgRX **placeholderTemplate** store exposed by the same package.
+
+The component only has 1 input and supports a *content value*.
+
+- The **id** input is required and will determine the unique identification for a specific placeholder component.
+- The *content value* (or child nodes), if provided, will be used by the placeholder component as default display when no template has been found.
+
+## How it works
+
+Based on the **id** provided to the placeholder component, it will register itself to the event coming from **placeholderTemplate** and will display the template corresponding to its ID in the store.
+
+> **note**: it is **strongly encouraged** to use the placeholder mechanism in concert with the [Rules Engine](../rules-engine/) following [this documentation](../rules-engine/how-to-use/placeholders.md).
+
+## How to define a placeholder template
+
+The placeholder template is defined in a JSON file following this JSON Schema [placeholder-template.schema.json](../../packages/@o3r/components/schemas/placeholder-template.schema.json).
+
+Example:
+
+```json
+{
+  "template": "<p>My fact : <%= myFact %></p>",
+  "vars": {
+    "myFact ": {
+      "value": "myFact ",
+      "type": "fact"
+    }
+  }
+}
+```
+
+Then the JSON File can be downloaded and dispatched to the store as follows:
+
+```typescript
+import { setPlaceholderTemplateEntityFromUrl } from '@o3r/components';
+
+const JsonFileUrl = 'url-to-the-json-file';
+
+store.dispatch(setPlaceholderTemplateEntityFromUrl({
+  call: (await fetch(JsonFileUrl)).json(),
+  id: 'my-placeholder-template',
+  resolvedUrl: JsonFileUrl,
+  url: JsonFileUrl
+}));
+```
+
+Or the object can be directly dispatched via the `setPlaceholderTemplateEntity` action.
```

Comment:
the internal mechanism explained here has changed with https://github.com/AmadeusITGroup/otter/commit/e88d6eb0d980ed34c09bca358399d04bf934e9f2